examples/illustrations/cell_site/src/cell_site-cdr.json

```diff
@@ -0,0 +1,29 @@
+[
+    {
+        "@id": "kb:9dcd8413-f681-4bc6-a66e-b70a7ecd3e4a",
+        "@type": "drafting:CapturedTelecommunicationsInformation",
+        "uco-core:hasFacet": [
+            {
+                "@type": "drafting:CapturedTelecommunicationsInformationFacet",
+                "drafting:captureCellSite": {
+                    "@id": "urn:example:cell-site-kb:cell-cite-204-16-1014-13399"
+                },
+                "observable:startTime": {
+                    "@type": "xsd:dateTime",
+                    "@value": "2021-07-29T01:23:02Z"
+                },
+                "observable:endTime": {
+                    "@type": "xsd:dateTime",
+                    "@value": "2021-07-30T02:10:50Z"
+                },
+                "drafting:partySubscription": {
+                    "@id": "kb:4b3cdcbd-6a31-462f-be9b-1ca2944c8876"
+                },
```

Comment:
The graph object referenced in `partySubscription` is not provided in this example.  Its English definition, and generic range of `ObservableObject`, also makes it difficult to understand what that object should be.  What clusters address properties?

I recommend deleting this property from the proposal.

Reply:
`partySubscription` has been removed.